bessctl/test_sugar.py

```diff
@@ -0,0 +1,100 @@
+import fnmatch
+import os
+import sugar
+import unittest
+
+
+class TestSugar(unittest.TestCase):
+    """
+    All scripts in conf/ will be dynamically added here as individual
```

Comment:
I would guess that shelling out will break it but I guess that can be fixed when you figure it out. Okay just curious, thought that you had figured out how to do it even with the shelling out :-)